documentation/modules/exploit/multi/http/mediawiki_syntaxhighlight.md

```diff
@@ -0,0 +1,30 @@
+## Vulnerable Application
+
+  Any MediaWiki installation with SyntaxHighlight version 2.0 installed & enabled. This extension ships with the AIO package of MediaWiki 1.27.x & 1.28.x. This issue was supposed to be fixed in MediaWiki version 1.28.1 and version 1.27.2. It appears that the fix was pushed to the git repository, but for some reason it was not included in the release packages.
+
+## Verification Steps
+
+  1. `use exploit/multi/http/mediawiki_syntaxhighlight`
+  2. `set RHOST <ip target site>`
+  3. `set TARGETURI <MediaWiki path>`
+  4. `set UPLOADPATH <writable path in web root>`
+  5. optionally set `RPORT`, `SSL`, and `VHOST`
+  6. `exploit`
+  7. **Verify** a new Meterpreter session is started
+
```

Comment:
Can you add an #Options section here as well.  I think the only option that needs to be mentioned is that UPLOADPATH needs to be writable, and the default `images` typically is.

Example: https://github.com/rapid7/metasploit-framework/blob/master/documentation/modules/exploit/linux/local/bpf_priv_esc.md#options